internal/sbom/spdx.go

```diff
@@ -10,10 +12,10 @@ import (
 )
 
 type SPDX struct{}
-type spdxLoader func(io.Reader) (*spdx.Document2_2, error)
+type SPDXLoader func(io.Reader) (*spdx.Document2_2, error)
```

Comment:
is this strictly needed? This makes the type public.

I think the rule is that the casing is consistent for acronyms (i.e. either always UPPERCASE or lowercase).

Reply:
Reverted this change